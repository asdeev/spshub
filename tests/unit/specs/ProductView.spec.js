import { shallowMount } from '@vue/test-utils';
import { expect } from 'chai';
import testMap from '~/tests/testMap.js';

describe('ProductView', () => {
  it('renders the component', () => {
    const wrapper = shallowMount(testMap.ProductView);

    expect(wrapper.html()).to.matchSnapshot();
  });
});
