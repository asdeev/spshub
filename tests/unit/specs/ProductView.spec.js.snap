// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductView renders the component 1`] = `
"<div> ProductView
</div>"
`;
